mdm/mdm/command.go

```diff
@@ -60,8 +60,11 @@ type Command struct {
 	ScheduleOSUpdateScan            *ScheduleOSUpdateScan
 	ActiveNSExtensions              *ActiveNSExtensions
 	RotateFileVaultKey              *RotateFileVaultKey
+	ActivationLockBypassCode        *ActivationLockBypassCode
 }
 
+type ActivationLockBypassCode struct{}
```

Comment:
https://macadmins.slack.com/archives/C19RTE0L9/p1551718526006700

See the conversation in MacAdmins slack. I don't believe we need the explicit type here or in the list above. Since the command takes no params. I think you can remove the changes in this file but keep the marhsalling code and the test command should still work

Reply:
I missed that one when I simplified the code. I updated it now.